Makefile.am

```diff
@@ -960,15 +934,6 @@ imap_imapd_LDADD = $(LD_SIEVE_ADD) $(LD_SERVER_ADD)
 imap_ipurge_SOURCES = imap/cli_fatal.c imap/ipurge.c imap/mutex_fake.c
 imap_ipurge_LDADD = $(LD_UTILITY_ADD)
 
-nodist_imap_libcyrus_imap_la_SOURCES = \
```

Comment:
Leave these as `nodist_` I think, these files don't need to be distributed.  If they were in EXTRA_DIST, that's the bug, not this